app/presenters/keyword_presenter.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class KeywordPresenter
```

Comment:
Where is this initialized and used 🤔 ?

Reply:
I got mixed up with the backend PR. Here's how it is used:
https://github.com/Goose97/google-scraper-ruby/pull/35/files#diff-894e2178295890bab1c6d5f69e7fb72e96a103559cf9a87c2c3cd8920bc282c4R9